core/templates/pages/blog-home-page/blog-home-page.component.html

```diff
@@ -114,20 +114,19 @@ <h4 class="d-flex flex-grow-1 search-heading">
           </h4>
           <form class="navbar-form oppia-search-bar-form pl-0 e2e-test-search-field" role="search">
             <div class="form-group d-flex"
-                 [ngClass]="{'flex-row-reverse': isSmallScreenViewActive()}">
+                 >
```

Comment:
Move this to the end of the previous line.

Reply:
Done.